contrib/inventory/spacewalk.py

```diff
@@ -57,12 +57,15 @@
     import simplejson as json
 
 base_dir  = os.path.dirname(os.path.realpath(__file__))
+default_ini_file = os.path.join(base_dir, "spacewalk.ini")
+
 SW_REPORT = '/usr/bin/spacewalk-report'
 CACHE_DIR = os.path.join(base_dir, ".spacewalk_reports")
 CACHE_AGE = 300 # 5min
-INI_FILE = os.path.join(base_dir, "spacewalk.ini")
+INI_FILE = os.path.expanduser(os.path.expandvars(os.environ.get("SPACEWALK_INI", default_ini_file)))
```

Comment:
Perhaps it could be SPACEWALK_CONFIG, and be consistent with other variables such as ANSIBLE_CONFIG, [COLINS_CONFIG](https://github.com/ansible/ansible/blob/devel/contrib/inventory/collins.py#L38).

Note that gce uses [GCE_INI_PATH](https://github.com/ansible/ansible/blob/devel/contrib/inventory/gce.py#L134), docker uses DOCKER_CONFIG_FILE, nsot uses NSOT_INVENTORY_CNFIG... 

Not sure what the core devs would prefer here, but perhaps it would make sense to be consistent with ANSIBLE_CONFIG.